site/_data/github-labels.yaml

```diff
@@ -0,0 +1,308 @@
+      target: both
+      addedBy: anyone
+    - color: "795548"
+      description: Denotes an issue or PR has remained open with no activity and has become stale.
+      name: lifecycle/stale
+      target: both
+      addedBy: anyone
+      previously:
+        - name: stale
+    - color: "F58C28"
+      description: Denotes an issue that is assigned and is being actively investigated by the issue owner.
+      name: lifecycle/investigating
+      target: both
+      addedBy: anyone
+    - color: "90A8E4"
+      description: Denotes an issue that has been triaged and determined to be valid.
+      name: lifecycle/accepted
+      target: both
+      addedBy: anyone
+
+# Priority.
+    - color: fef2c0
+      description: Lowest priority. Possibly useful, but not yet enough support to actually get it done. # These are mostly place-holders for potentially good ideas, so that they don't get completely forgotten, and can be referenced /deduped every time they come up.
+      name: priority/awaiting-more-evidence
+      target: both
+      addedBy: anyone
+    - color: fbca04
+      description: Higher priority than priority/awaiting-more-evidence. # There appears to be general agreement that this would be good to have, but we may not have anyone available to work on it right now or in the immediate future. Community contributions would be most welcome in the mean time (although it might take a while to get them reviewed if reviewers are fully occupied with higher priority issues, for example immediately before a release).
+      name: priority/backlog
+      target: both
+      addedBy: anyone
+    - color: e11d21
+      description: Highest priority. Must be actively worked on as someone's top priority right now. # Stuff is burning. If it's not being actively worked on, someone is expected to drop what they're doing immediately to work on it. Team leaders are responsible for making sure that all the issues, labeled with this priority, in their area are being actively worked on. Examples include user-visible bugs in core features, broken builds or tests and critical security issues.  name: priority/critical-urgent
+      name: priority/critical-urgent
+      target: both
+      addedBy: anyone
+      previously:
+        - name: "p0 - Hair On Fire"
+    - color: eb6420
+      description: Must be staffed and worked on either currently, or very soon, ideally in time for the next release.
+      name: priority/important-soon
+      target: both
+      addedBy: anyone
+      previously:
+        - name: "p1 - Important"
+    - color: eb6420
+      description: Important over the long term, but may not be staffed and/or may need multiple releases to complete.
+      name: priority/important-longterm
+      target: both
+      addedBy: anyone
+      previously:
+        - name: "p2 - Long-term Important"
+
+# Size.
+    - color: ee9900
+      description: Denotes a PR that changes 100-499 lines, ignoring generated files.
+      name: size/L
```

Comment:
How do we determine the application of these labels? Using personal experience?

Reply:
I don't have a proposal for a rule here. These are widely used in Kubernetes and it seemed useful to be able to quickly see a guide to the size/scope of an issue when we do planning. I'm fine leaving it to the judgement of maintainers and gathering experience on how useful this is.